app/api/authentication.js

```diff
@@ -0,0 +1,66 @@
+
```

Comment:
We should think about making this file an external module one day for reuse in other projects (when day we need it comes)